### PR TITLE
rishabhrathore055

### PR DESCRIPTION
**Pull request by rishabhrathore055**
#445

### CODE REVIEW

File: data.json [hunk 1]
```diff
@@ -760,5 +760,11 @@
     "githubUsername": "anotherwebguy",
     "imageUrl": "https://user-images.githubusercontent.com/66346161/114270825-18c90d80-9a2c-11eb-9173-1c072e90ac27.png",
     "issueNumber": 436
+  },
+   {
```
Comment (schmelto) [inline data.json:764]:
```suggestion
  {
```
please make this line up with the other braces

File: data.json [hunk 2]
```diff
@@ -760,5 +760,11 @@
...
+    "name": "Rishabh Rathore",
+    "githubUsername": "Rishabhrathore055",
+    "imageUrl": "https://user-images.githubusercontent.com/77366500/115426452-234e8880-a21e-11eb-9e2d-c508acdb921f.jp",
```
Comment (schmelto) [inline data.json:767]: take a look at your image link there is a missing `p` for `jpg`.

```suggestion
    "imageUrl": "https://user-images.githubusercontent.com/77366500/115426452-234e8880-a21e-11eb-9e2d-c508acdb921f.jpg",
```

**Review by schmelto — CHANGES_REQUESTED** [review]
can you change the description from #445 to resolve #445 or fixes #445 that the issue is automatically closed on PR merge 🚀

**Review by schmelto — APPROVED** [review]
LGTM

**Review by AllanRegush — APPROVED** [review]
Looks Good :+1: Thank you :smile:

**Comment by AllanRegush** [comment]
Thanks for reviewing @schmelto

**Comment by rishabhrathore055** [comment]
ThankYou So Much

**Pull request merged**